test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.Order.html

```diff
@@ -41,6 +41,9 @@
                 <option value="Credit">Credit</option>
```

Comment:
CI builds are failing because you didn't update `HtmlGenerationWebSite.HtmlGeneration_Home.Order.Encoded.html`.